src/test/resources/Features/surveillanceComplaints.feature

```diff
@@ -0,0 +1,70 @@
+@Regression
+@complaints
+Feature: Verify complaints can be added
+         OCD-2912-Create a basic complaint, edit a complaint, delete a complaint
+         OCD-2913-Associate a complaint with a listing
+         OCD-2914-Associate a complaint with criteria
+         OCD-2915-Associate a complaint with surveillance
+
+Scenario Outline: Add a complaint
+    Given I'm logged in as "ROLE_ACB"
+    And I am on complaints page
+    When I click Add New Complaint link
+    Then "Add Complaint" form should open to add new complaint
+    When I enter complaint data in fields: "<ONC-ACB>" "<Received Date>" "<Closed Date>" "<ONC-ACB Complaint ID>" "<ONC Complaint ID>" "<Complainant Type>" "<Summary>"
+    And I click save button
+    Then a new complaint with given "<ONC-ACB Complaint ID>" should be added to CHPL 
+Examples:
+    |ONC-ACB|Received Date|Closed Date|ONC-ACB Complaint ID|ONC Complaint ID|Complainant Type|Summary|
+    |Drummond Group|07/12/2019|07/12/2019|ONCACB001|ONC001|Developer|Test Summary|
+    
+Scenario: Edit a complaint
+    Given I'm logged in as "ROLE_ACB"
+    And I am on complaints page
+    When I click Edit complaint for ONC-ACB Complaint ID "ONCACB001"
```

Comment:
This step assumes the previous one has already run. I don't have a better option for this right now; we need a "real" complaint to edit, so will not block this PR based on this, but want to update it once we actually have real data

Reply:
I thought about adding a new complaint and then doing some operation on it but the also thought it probably wasn't necessary. I agree with you that there shouldn't be dependency between scenarios. I'll make that change for this & delete scenario (next comment).